packages/prompts/src/index.ts

```diff
@@ -12,13 +12,14 @@ import {
 	TextPrompt,
 	block,
 	isCancel,
+	updateSettings as coreUpdateSettings,
+	type ClackSettings as CoreClackSettings,
```

Comment:
I think it's okay to keep the same export name here?

Reply:
typescript yells